firebase-vscode/package.json

```diff
@@ -0,0 +1,122 @@
+{
+  "name": "firebase-vscode",
+  "displayName": "firebase-vscode",
+  "publisher": "firebase",
+  "description": "",
+  "version": "0.0.7",
+  "engines": {
+    "vscode": "^1.69.0"
+  },
+  "repository": "https://github.com/firebase/firebase-tools",
+  "sideEffects": false,
+  "categories": [
+    "Other"
+  ],
+  "activationEvents": [
+    "onStartupFinished"
+  ],
+  "main": "./dist/extension.js",
+  "contributes": {
+    "configuration": {
+      "title": "Firebase VS Code Extension",
+      "properties": {
+        "firebase-vscode-extension.firebaseRcFolder": {
+          "type": "string",
+          "default": "",
+          "description": "Firebase RC folder"
+        }
+      }
+    },
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "firebase2",
```

Comment:
All these "firebase2"s were supposed to be temporary - it was conflicting with the hackweek project embedded into Monospace which used the same ID. Need to revert these back to `firebase`.